core/tests/protractor/learnerDashboard.js

```diff
@@ -39,210 +42,309 @@ var SubscriptionDashboardPage =
+    explorationEditorMainTab.setInteraction('Continue');
+    var responseEditor = explorationEditorMainTab.getResponseEditor('default');
+    responseEditor.setDestination('Second', true, null);
+    explorationEditorMainTab.moveToState('Second');
+    explorationEditorMainTab.setContent(forms.toRichText(
+      'So what can I tell you?'));
+    explorationEditorMainTab.setInteraction('MultipleChoiceInput', [
+      forms.toRichText('How do your explorations work?'),
+      forms.toRichText('What can you tell me about this website?'),
+      forms.toRichText('How can I contribute to Oppia?'),
+      forms.toRichText('Those were all the questions I had!')
+    ]);
+    explorationEditorMainTab.addResponse(
+      'MultipleChoiceInput', null, 'End Card', true, 'Equals',
+      'Those were all the questions I had!');
+    responseEditor = explorationEditorMainTab.getResponseEditor('default');
+    responseEditor.setFeedback(forms.toRichText('I do not know!'));
+    explorationEditorMainTab.moveToState('End Card');
+    explorationEditorMainTab.setContent(
+      forms.toRichText('Congratulations, you have finished!'));
+    explorationEditorMainTab.setInteraction('EndExploration');
+    explorationEditorPage.navigateToSettingsTab();
+    explorationEditorSettingsTab.setTitle('About Oppia');
+    explorationEditorSettingsTab.setCategory('Algorithm');
+    explorationEditorSettingsTab.setObjective('Learn more about Oppia');
+    explorationEditorSettingsTab.setLanguage('English');
+    explorationEditorPage.saveChanges();
+    workflow.publishExploration();
+  };
+
+  it('displays completed explorations', function() {
+    users.createUser('originalCreator@learnerDashboard.com',
+      'originalCreator');
+    users.createUser('learner@learnerDashboard.com',
+      'learnerlearnerDashboard');
+    users.createAdmin('inspector@learnerDashboard.com', 'inspector');
+    users.login('originalCreator@learnerDashboard.com');
+    // Create first exploration named 'About Oppia'
+    createAboutOppiaExploration();
+    // Create a second exploration named 'Dummy Exploration'.
+    workflow.createAndPublishExploration(
+      'Dummy Exploration',
+      'Algebra',
+      'To learn about Algebra!',
+      'English'
+    );
+    users.logout();
     users.login('learner@learnerDashboard.com');
-
-    // Play an exploration and leave it in between. It should be added to the
-    // 'In Progress' section.
-    general.openPlayer('3');
-    explorationPlayerPage.submitAnswer('Continue', null);
-    browser.ignoreSynchronization = true;
-    learnerDashboardPage.get();
-    general.acceptAlert();
-    browser.ignoreSynchronization = false;
-    browser.waitForAngular();
-    libraryPage.expectExplorationToBeVisible('Root Linear Coefficient Theorem');
-
-    // Play an exploration completely. It should be added to the 'Completed'
-    // section.
-    general.openPlayer('14');
+    // TODO(hoangviet1993): Leave the exploration in between.
```

Comment:
Are we planning to resolve this TODO before merge?

Reply:
TODO is finished!